pkgs/by-name/vo/voicevox-core/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  autoPatchelfHook,
+}:
+
+let
+  platformInfoTable = {
+    "x86_64-linux" = {
+      id = "linux-x64";
+      hash = "sha256-/PD5e0bWgnIsIrvyOypoJw30VkgbOFWV1NJpPS2G0WM=";
+    };
+    "aarch64-linux" = {
+      id = "linux-arm64";
+      hash = "sha256-zfiorXZyIISZPXPwmcdYeHceDmQXkUhsvTkNZScg648=";
+    };
+    "x86_64-darwin" = {
+      id = "osx-x64";
+      hash = "sha256-cdNdV1fVPkz6B7vtKZiPsLQGqnIiDtYa9KTcwSkjdJg=";
+    };
+    "aarch64-darwin" = {
+      id = "osx-arm64";
+      hash = "sha256-cdNdV1fVPkz6B7vtKZiPsLQGqnIiDtYa9KTcwSkjdJg=";
```

```suggestion
      hash = "sha256-Z1dq2t/HBQulbPF23ZCihOrcZHMpTXEQ6yXKORZaFPk=";
```

Accidental duplicate of x86_64-darwin

---

Ah, I missed one, thanks.

---

It builds on aarch64-Darwin now! 🎉